OpenRA.Mods.Common/Traits/Air/TargetableAircraft.cs

```diff
@@ -35,7 +35,8 @@ public override string[] TargetTypes
 		{
 			get
 			{
-				return (self.CenterPosition.Z > 0) ? info.TargetTypes
+				return IsTraitDisabled ? new string[] { }
+					: (self.CenterPosition.Z > 0) ? info.TargetTypes
 					: info.GroundedTargetTypes;
```

Comment:
Indentation here isn't consistent. Some parentheses around the outer `else` would be good for readability, and maybe have in on one line.